src/appleseed/foundation/meta/tests/test_aabb.cpp

```diff
@@ -536,4 +599,15 @@ TEST_SUITE(Foundation_Math_AABB)
         EXPECT_FALSE(bbox1 != bbox2);
         EXPECT_TRUE(bbox1 != bbox3);
     }
+
+    TEST_CASE(TestOneByOneOnIntegralAABB)
```

Comment:
Maybe `TestExtentOn1x1x1IntegralAABB`? Probably best moved next to the other tests for `extent()`.